### DISCUSSION TITLE
Add autocomplete "new-password" for the creation of users

### DISCUSSION THREAD

**Discussion by Silbad** [directus/directus#5522] [Feature Requests]
Hello,
When creating a new user, the autocomplete feature is enabled by default. Is it possible to disable it?
=> add an attribut autocomplete="new-password" to the input type password.
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
Thanks

**Comment by rijkvanzanten** [comment]
That's the thing.. It's not. It's disabled by default:

https://github.com/directus/directus/blob/c6927bb4e28959377af0ec7b7b54fc67374ba8fd/app/src/components/v-input/v-input.vue#L133-L136

However, some browsers, and most notably password managers started ignoring the `autocomplete` attribute, making the spec completely useless! 😠

**Reply by ernst86913** [reply]
Especially when opening a user record its extremely annoying that password managers create a mess with the user record.

Rijk - what about making those fields hidden in the default user record and add a "Change Password" action button to avoid this issue?

**Reply by rijkvanzanten** [reply]
> Rijk - what about making those fields hidden in the default user record and add a "Change Password" action button to avoid this issue?

That doesn't solve the problem for any other masked password type inputs 🤔 We'll have to come up with a way to do a masked input without using `type="password"` on the input. If only `-webkit-text-security` ever made it into the spec...

The next best thing might be to do it in JS in the v-input itself, where it'll keep track of the actual value, but only renders `•` characters based on value.length 🤔

**Reply by nickrum** [reply]
@rijkvanzanten have you actually tried using `autocomplete="new-password"`? At least [mdn](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#preventing_autofilling_with_autocompletenew-password) suggests that it is respected by modern browsers and their password managers.

**Reply by rijkvanzanten** [reply]
@nickrum Wouldn't that trigger things like "Use suggested password: xxx" still? 🤔

**Reply by rijkvanzanten** [reply]
But yeah, it beats the current situation for sure 🙂

**Reply by nickrum** [reply]
It probably would... But it's also kind of what you expect from using a password manager.

**Comment by nickrum** [comment]
This has been addressed in #12465 👍
